portal/server/controllers/utilisation-report-service/utilisation-report-upload/index.js

```diff
@@ -117,26 +125,28 @@ const getReportConfirmAndSend = async (req, res) => {
 };
 
 const postReportConfirmAndSend = async (req, res) => {
-  const { userToken, utilisation_report: utilisationReport } = req.session;
+  const { userToken, utilisation_report: utilisationReport, user } = req.session;
```

Comment:
Discussed: need to change the `utilisation_report: utilisationReport` part of this

Reply:
fwiw i've done this on my branch with a full rename of utilisation_report to utilisationReport everywhere it's in the session